spec/features/form/accessible_autocomplete_spec.rb

```diff
@@ -79,6 +59,32 @@
     end
   end
 
+  context "when searching schemes" do
+    let(:scheme) { FactoryBot.create(:scheme, owning_organisation_id: case_log.created_by.organisation_id, primary_client_group: "Q", secondary_client_group: "P") }
+
+    before do
+      FactoryBot.create(:location, scheme:, postcode: "W6 0ST")
+      FactoryBot.create(:location, scheme:, postcode: "SE6 1LB")
+      case_log.update!(needstype: 2)
```

Comment:
Any reason not to just have this in the FactoryBot create?

Reply:
What do you mean?